riscv/interactive.cc

```diff
@@ -497,11 +502,17 @@ void sim_t::interactive_until(const std::string& cmd, const std::vector<std::str
   bool cmd_until = cmd == "until" || cmd == "untiln";
 
   if (args.size() < 3)
-    return;
+   throw trap_interactive();
```

Comment:
indentation doesn't match.  Since this is coming up frequently, you probably need to fix your editor to make the tab stop 2 spaces.